### PR TITLE
Copy dependabot.yml from celestiaorg/celestia-core

### PR DESCRIPTION
**Pull request by jbowen93**
Resolves https://github.com/celestiaorg/celestia-node/issues/64

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - fadeev
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "11:00"
+    open-pull-requests-limit: 10
+    reviewers:
+      - melekes
+      - tessr
```
Comment (renaynay) [inline .github/dependabot.yml:25]:
```suggestion
      - Wondertan
      - renaynay
```

**Review by renaynay — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (liamsi) [reply .github/dependabot.yml:25]: I think the CODEOWNERS overwrites this anyways but yeah doesn't make sense and we should update it.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    directory: "/"
+    schedule:
+      interval: daily
+      time: "11:00"
+    open-pull-requests-limit: 10
+    reviewers:
+      - Wondertan
+      - renaynay
+    labels:
+      - T:dependencies
```
Comment (Wondertan) [inline .github/dependabot.yml:28]: Pls new line her

**Review by Wondertan — CHANGES_REQUESTED** [review]
Can we make the schedule weekly pls? I don't want to receive daily notifications pls

File: .github/dependabot.yml [hunk 3]
```diff
@@ -3,13 +3,15 @@ updates:
...
     open-pull-requests-limit: 10
   - package-ecosystem: npm
     directory: "/docs"
     schedule:
-      interval: daily
+      interval: weekly
+      day: monday
       time: "11:00"
     open-pull-requests-limit: 10
     reviewers:
```
Comment (Wondertan) [inline .github/dependabot.yml]: One last thing, we don't need a npm

**Review by Wondertan — APPROVED** [review]

**Review by renaynay — APPROVED** [review]

**Pull request merged**